src/rehype-confluence.ts

```diff
@@ -15,22 +15,36 @@ const CLASSNAME_MAPPER = {
   confluenceTh: 'border-th',
 }
 
+/**
+ * remove jira element
+ */
+export function isJiraElement(node: Element | ElementContent): boolean {
+  if (node.type === 'text') {
+    return node.value.toLowerCase().includes('jira')
+  }
+  if (node.type === 'element' && node.children.length > 0) {
```

Comment:
```suggestion
  if (node.type === 'element') {
```

好像没必要判断 length，为 0 的话 `.some` 返回就是 false